initfiles/componentfiles/configxml/sparkThor.xsd

```diff
@@ -206,27 +145,13 @@
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
-            <xs:attribute name="SPARK_WORKER_PORT" type="xs:nonNegativeInteger" default="7070">
+            <xs:attribute name="SPARK_WORKER_PORT" type="xs:nonNegativeInteger" default="7071">
```

Comment:
this is b/c dali tends to run on 7070?

Reply:
Yes.  And this gets written out to the spark-env.sh file so all the workers will be able to see the new default port.